package.json

```diff
@@ -17,6 +17,7 @@
     "jobs:debug": "npm run build && node --inspect=0 ./built/jobs/index.js",
     "web": "node ./built/web/index.js",
     "web:dev": "node -r dotenv/config ./built/web/index.js",
+    "web:nosync": "node -r dotenv/config ./built/web/index.js --noSync",
```

Comment:
I would prefer 'keep-db' over 'no-sync'

Reply:
okay, for some reason isCommandArg is not working with ('--noSync'), do you have an idea what is different wit '--noPDF'?

Reply:
I think isCommandArg is buggy, but it has worked, because --noPDF was the only possible param